core/functions/fn_addItemOrg.sqf

```diff
@@ -0,0 +1,184 @@
+/*
+ * Author: Olsen
+ *
+ * Add item to unit.
+ *
+ * Arguments:
+ * 0: unit to add item to <object>
+ * 1: name of loadout <string>
+ * 2: item classname <string>
+ * 3: amount <number> (optional)
+ * 4: container ("uniform"/"vest"/"backpack") <string> (optional)
+ *
+ * Return Value:
+ * nothing
+ *
+ * Public: No
+ */
+
+#include "..\script_macros.hpp"
+
+params ["_unit", "_loadoutType", "_item", ["_amount", 1, [1]], ["_position", "", [""]]];
+
+if !([_item, _unit] call FUNC(checkClassname)) exitWith {};
+
+if !(_position isEqualTo "") then {
+	_position = toLower _position;
+};
+
+for "_x" from 1 to _amount do {
+	private _success = false;
+	private _parents = [configFile >> "CFGweapons" >> _item, true] call BIS_fnc_returnParents;
+	private _type = (_item call BIS_fnc_itemType) select 1;
+
+	if (_position isEqualTo "none") then {
+		if (!_success && {"Rifle" in _parents}) then {
+			if (primaryWeapon _unit isEqualTo "") then {
+				_unit addWeaponGlobal _item;
+				_success = true;
+			};
+		};
+		if (!_success && {"Pistol" in _parents}) then {
+			if (handgunWeapon _unit isEqualTo "") then {
+				_unit addWeaponGlobal _item;
+				_success = true;
+			};
+		};
+		if (!_success && {"Launcher" in _parents}) then {
+			if (secondaryWeapon _unit isEqualTo "") then {
+				_unit addWeaponGlobal _item;
+				_success = true;
+			};
+		};
+		if (!_success && {_type in ["Map", "GPS", "Compass", "Watch", "NVGoggles"]}) then {
+			if ([_unit, _type] call FUNC(CanLinkItem)) then {
+				_unit linkItem _item;
+				_success = true;
+			};
+		};
+		if (!_success && {_type isEqualTo "uniform"}) then {
```

Comment:
_type is going to return a leading-capitalized letter for all types (per https://community.bistudio.com/wiki/BIS_fnc_itemType and local testing)